x-pack/platform/plugins/shared/lock-manager/kibana.jsonc

```diff
@@ -0,0 +1,17 @@
+{
+  "type": "plugin",
+  "id": "@kbn/lock-manager",
```

Comment:
@elastic/kibana-operations, do you anticipate any issues of _the package name_ of a plugin not having the suffix `-plugin`?

Reply:
I'm happy to add the plugin suffix.

Reply:
Is there any tooling for generating a plugin? Would be great if I could just `node scripts/create-plugin "lockManger"` and then it would create a minimal boilerplate

Reply:
afaik, we don't currently have any requirements on the naming - however, it might help in the future to categorize plugins/packages

Reply:
It's in the backlog https://github.com/elastic/kibana/issues/162065

Reply:
Any idea what this error is about?

```
 node scripts/type_check.js
ERROR UNHANDLED ERROR
ERROR Error: invalid kbn_references in @kbn/test-suites-xpack: @kbn/lock-manager does not point to another TS project
          at flatMap (ts_project.ts:313:15)
          at Array.flatMap (<anonymous>)
          at TsProject.getKbnRefs (ts_project.ts:299:39)
          at createTypeCheckConfigs (run_type_check_cli.ts:55:27)
          at description (run_type_check_cli.ts:109:27)
          at run.ts:80:10
          at withProcRunner (with_proc_runner.ts:30:5)
          at run (run.ts:78:5)
```

Reply:
Figured it out https://github.com/elastic/kibana/pull/219220/commits/c9338c0a9916e0f417c13ecba11b0ff092156b1c